examples/v1alpha1/bitbucket/README.md

```diff
@@ -0,0 +1,40 @@
+## Bitbucket EventListener
+
+Creates an EventListener that listens for Bitbucket webhook events.
+
+### Try it out locally:
+
+1. To create the Bitbucket trigger and all related resources, run:
+
+   ```bash
+   kubectl apply -f examples/v1alpha1/bitbucket/
```

Comment:
bikeshed: should we assume that commands are relative to their directory? (i.e. `kubectl apply -f .`)?

We're using a mix of repo root relative and example directory relative paths in examples. We should probably standardize on one (I think example directory relative makes the most sense).

Reply:
yeah that was weird. Switched them all to be relative to their directory instead of repo root.